helm/sematic-server/templates/ingress.yaml

```diff
@@ -9,56 +9,63 @@ metadata:
   labels:
     {{- include "sematic-server.labels" . | nindent 4 }}
   annotations:
+    {{- if .Values.gcp.enabled }}
+    kubernetes.io/ingress.class: {{ .Values.ingress.class_name }}
+    {{- if .Values.ingress.force_ssl }}
+    networking.gke.io/v1beta1.FrontendConfig: {{ .Release.Name }}-https-redirect
+    {{- end }}
+    {{- else }}
     {{- if .Values.ingress.force_ssl }}
```

Comment:
Should this section then check for an equivalent new `.Values.aws.enabled` condition in order to improve readability and structuring of the configuration?